### ISSUE TITLE
Jupyter crashes

### ISSUE THREAD

**Issue by smith6jt-cop** [peng-lab/BaSiCPy#157]
Hello,

Trying this code from your notebooks as the standard autotune approach does not appear to give the best results.  However, after only a minute and a few iterations, jupyter kernel restarts.  Running on Windows 10, 32 core, 64 GB RAM.

I tried all options for fitting_mode, resize_mode, and max iterations, but still crashing.  Can you help get this going or suggest a method to find optimal parameters?

flatfields_wsi = []
darkfields_wsi = []
records = []

for smoothness_flatfield in [0.1, 0.5, 1, 5, 10]:
    for smoothness_darkfield in [0, 0.1, 0.5, 1, 5, 10]:
        for sparse_cost_darkfield in tqdm(np.array([0, 0.1, 0.5, 1, 5, 10]) * 0.01):
            basic = BaSiC(
                fitting_mode='approximate',
                max_iterations = 100,
                max_reweight_iterations = 5,
                max_reweight_iterations_baseline= 3,
                max_workers = 20,
                resize_mode = 'skimage_dask',
                smoothness_flatfield=smoothness_flatfield,
                smoothness_darkfield=smoothness_darkfield,
                sparse_cost_darkfield=sparse_cost_darkfield,
                get_darkfield=True,
            )
            try:
                basic.fit(im_array)
                flatfields_wsi.append(basic.flatfield)
                darkfields_wsi.append(basic.darkfield)
                records.append(
                    {
                        "smoothness_flatfield": smoothness_flatfield,
                        "smoothness_darkfield": smoothness_darkfield,
                        "sparse_cost_darkfield": sparse_cost_darkfield,
                    }
                )
            except RuntimeError:
                pass
conditions_df = pd.DataFrame.from_records(records)

**Comment by smith6jt-cop** [comment]
Link to images: https://drive.google.com/drive/folders/1PqpMqqZCASW0VqwwaEf_MENy9wqJnzAQ?usp=sharing

**Comment by yuliu96** [comment]
Hi, for BaSiCPy, can you primarily try without darkfield now (so get_darkfield=False)? we will elaborate the effects of darkfield estimation in our later manuscript.

**Comment by smith6jt-cop** [comment]
Yes it does work more consistently that way; however, the darkfield correction improves greatly some images so I would like to still be able to use it.  I look forward to your manuscript, but in the meantime would you please briefly describe the problem with the darkfield function causing the jupyter kernel to fail?  I would greatly appreciate your help.  Thank you in advance.
